image-config.yaml

```diff
@@ -9,9 +9,5 @@ spec:
   builders:
     jdk7:
       tag: jdk:7,maven:3.8,ant:1.9.16
-    jdk8:
-      tag: jdk:8,maven:3.8,gradle:8.3;8.0.2;7.4.2;6.9.2;5.6.4;4.10.3,sbt:1.8.0,ant:1.10.13
-    jdk11:
-      tag: jdk:11,maven:3.8,gradle:8.3;8.0.2;7.4.2;6.9.2;5.6.4;4.10.3,sbt:1.8.0,ant:1.10.13
     jdk17:
-      tag: jdk:17,maven:3.8,gradle:8.3;8.0.2;7.4.2;6.9.2,sbt:1.8.0,ant:1.10.13
+      tag: jdk:8,11,17,maven:3.8.8,3.9.5,gradle:8.3;8.0.2;7.4.2;6.9.2,sbt:1.8.0,ant:1.10.13
```

Comment:
The previous separator between versions used `;` so should this be
```
jdk:8;11;17,maven:3.8.8;3.9.5,gradle:8.3;8.0.2;7.4.2;6.9.2,sbt:1.8.0,ant:1.10.13
```
?

Reply:
oops, my mistake, I was in a hurry